plugins/globalias/globalias.plugin.zsh

```diff
@@ -4,8 +4,11 @@ globalias() {
    zle self-insert
 }
 zle -N globalias
-bindkey -e " " globalias
-bindkey -v " " globalias
-bindkey -e "^ " magic-space           # control-space to bypass completion
-bindkey -v "^ " magic-space
-bindkey -M isearch " " magic-space # normal space during searches
+
+# space expands all global aliases
```

Comment:
Since `zle expand-word` is also used, this needs to be changed to `all aliases, including global`. I just tried it and every alias is expanded. That should also written in the README.